packages/pixeloven/cli-addon-storybook/src/commands/story.ts

```diff
@@ -1,32 +1,36 @@
-import { AddonStorybookRunContext, StorybookExecutionType } from "../types";
+import { AddonStorybookToolbox, StorybookExecutionOptionTypes, StorybookExecutionType } from "../types";
 
 export default {
     alias: ["--story", "-s"],
     name: "story",
-    run: async (context: AddonStorybookRunContext) => {
-        const { parameters, pixelOven, storybook } = context;
-        if (!parameters.first) {
-            pixelOven.invalidArgument("Must provide a story run type.");
+    run: async (toolbox: AddonStorybookToolbox) => {
+        const { parameters, pixelOven, storybook } = toolbox;
+        const task = parameters.first;
+        if (!task) {
+            pixelOven.invalidArgument("Must provide a task for Storybook to run.");
+            pixelOven.exit("Storybook", 1);
             return;
         }
-        if (!StorybookExecutionType.hasOwnProperty(parameters.first)) {
-            pixelOven.invalidArgument("Invalid run type provided.");
+        if (!StorybookExecutionType.hasOwnProperty(task)) {
+            pixelOven.invalidArgument(`Available Storybook tasks are "build" or "start".`, task);
+            pixelOven.exit("Storybook", 1);
             return;
         }
-        switch (parameters.first) {
+        /**
+         *  @todo: add "help" argument that prints available tasks and options
+         */
+        switch (task) {
             case "build":
             case "start": {
-                const argList = pixelOven.getArgList(
-                    parameters.first,
-                    parameters,
-                    {
-                        offset: 1,
-                        type: "withOptions",
-                    },
-                );
+                Object.keys(parameters.options).forEach(option => {
+                    if (!StorybookExecutionOptionTypes.hasOwnProperty(option)) {
+                        pixelOven.invalidArgument(`Available options for "${task}" are "--outputDir", "--port", or "--noQuiet"`, `--${option}`);
```

Comment:
Nice I like this. I think in the future we should have a way to build this list some how. It would be helpful for doing --help or something.

Reply:
Yeah, I thought about iterating over the enum values, but we'd probably want to make a helper for that so we can repeat the language consistently across the CLIs